src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -770,6 +770,12 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
       else View.Empty
     }
 
+  /** Alias for `prependAll` */
+  def concat[B >: A](suffix: IterableOnce[B]): CC[B] = appendAll(suffix)
```

Comment:
Yes, I initially did this but then removed the inlining to allow inheritors to provide a more optimal implementation of concat directly if they wanted to, but I'll make it `final` (and add `@inline` as well of Dotty allows it).